src/main/java/org/springframework/samples/petclinic/owner/PetController.java

```diff
@@ -73,6 +73,22 @@ public String initCreationForm(Owner owner, ModelMap model) {
 		return VIEWS_PETS_CREATE_OR_UPDATE_FORM;
```

Comment:
The file has increased in 16 lines.
<!-- lookout footnote separator -->
_Comment made by the analyzer Dummy._